src/test/a11y/tests/apply.js

```diff
@@ -65,9 +65,19 @@ const apply = [
     })
   },
   {
-    url: URLS['POSTCODE'],
+    url: URLS['MANUAL_ADDRESS'],
```

Comment:
Why have we moved manual address before postcode lookup? Not that it makes any functional difference at this point, but in terms of page flow, you should only be able to get to manual address after postcode lookup...

Reply:
Because you can't link to a page further in the flow. Our state machine will redirect you.